### DISCUSSION TITLE
Can we make it support other type of datframes to make it faster ?

### DISCUSSION THREAD

**Discussion by RBouallegue** [kernc/backtesting.py#745] [Ideas]
Maybe we can use VAEX  type to make the running of the backtesting library faster for large data frames

**Comment by kernc** [comment]
The first performance improvement on a platform like Windos might be made by reenabling multiprocessing:
https://github.com/kernc/backtesting.py/blob/65f54f6819cac5f36fd94ebf0377644c62b4ee3d/backtesting/backtesting.py#L1353-L1380
I welcome your investigation.
